reference/tools/data-migration/features/online-ddl-scheme.md

```diff
@@ -140,16 +142,21 @@ pt-osc 主要涉及的 SQL如下：
 -- 1.
```

Comment:
It is recommended to split the SQL block into several blocks according to the `1, 2, 3, 4` number. Otherwise, the large block containing multiple separate SQL statements might not be user-friendly. Thanks！

Reply:
Updated. PTAL, thanks~